app/handlers/karma.py

```diff
@@ -60,12 +82,18 @@ async def get_me(message: types.Message, chat: Chat, user: User, config: Config)
         "user {user} ask his karma in chat {chat}", user=user.tg_id, chat=chat.chat_id
     )
     uk, number_in_top = await get_me_chat_info(chat=chat, user=user)
-    msg = await message.reply(
+    bot_reply = await message.reply(
         f"Ваша карма в данном чате: <b>{uk.karma:.2f}</b> ({number_in_top})",
         disable_web_page_preview=True,
     )
-    asyncio.create_task(delete_message(msg, config.time_to_remove_temp_messages))
-    asyncio.create_task(delete_message(message, config.time_to_remove_temp_messages))
+
+    asyncio.create_task(
+        cleanup_command_dialog(
+            bot_message=bot_reply,
+            delete_bot_reply=True,
+            delay=config.time_to_remove_temp_messages,
```

Comment:
Do we have that key in config?

Reply:
https://github.com/bomzheg/KarmaBot/blob/master/app/models/config/__init__.py#L27